### DISCUSSION TITLE
How to evaluate functions for stability derivatives such as clq, cmzq or clqdot in unsteady calculations?

### DISCUSSION THREAD

**Discussion by hntter** [mdolab/adflow#292] [Q&A] [answered]
Hi everyone,

I noticed there are internal mapping for cost functions in adflow flow, which I think the included variables could be used in evalFuncs. I have tried an unsteady case with qmode=True. However after finishing the calculations for 2 periods, the dynamic derivatives I would like to evaluate such as clq, cmzq or clqdot are still equal to 0. 
May I ask how can I evaluate dyanmic derivatives in an unsteady case?

Best regares!

**Comment by sseraj** [comment, marked as answer]
The time spectral stability derivatives functionals are currently broken. This is mentioned in the code, but I just created #295 to keep track of this. We do not have immediate plans to fix this.
